frontend/src/metabase/visualizations/click-actions/actions/CombineColumns/CombineColumns.tsx

```diff
@@ -0,0 +1,58 @@
+import { t } from "ttag";
+
+import { CombineColumns } from "metabase/query_builder/components/expressions/CombineColumns";
+import type { LegacyDrill } from "metabase/visualizations/types";
+import type { ClickActionPopoverProps } from "metabase/visualizations/types/click-actions";
+import * as Lib from "metabase-lib";
+
+export const CombineColumnsAction: LegacyDrill = ({ question, clicked }) => {
+  const { isEditable } = Lib.queryDisplayInfo(question.query());
+
+  if (
+    !clicked ||
+    clicked.value !== undefined ||
+    !clicked.columnShortcuts ||
+    clicked?.extraData?.isRawTable ||
```

Comment:
```suggestion
    clicked.extraData?.isRawTable ||
```

I think you already checked that clicked exists on the prev steps